app/elements/bar-zoomable-chart/bar-zoomable-chart.html

```diff
@@ -0,0 +1,316 @@
+<link rel='import' href='../../bower_components/polymer/polymer.html'>
+<script src='../../bower_components/d3/d3.min.js' charset='utf-8'></script>
+<dom-module id='bar-zoomable-chart'>
+    <style>
+        :host {
+            display: block;            
+        }
+        #barChart {
+            width:100%;
+            height: 400px;
+        }
+        
+        .axis {
+          font-size: 11px;
+          fill: var(--bar-zoomable-chart-axis-color, --secondary-text-color);
+        }
+
+        .overview {
+            margin-top: 300px;
+        }
+
+        .axis line, .axis path {
+          fill: none;
+          stroke: rgba(0,0,0,var(--dark-divider-opacity));
+          shape-rendering: crispEdges;
+        }   
+        .brush .extent {
+          stroke: #fff;
+          fill-opacity: .125;
+          shape-rendering: crispEdges;
+        }
+    </style>
+    <template>
+    <div class='layout horizontal'>
+      <div id='chartContainer' class='flex auto'>
+        <svg id='barChart'></svg>
+      </div>
+    </div>
+
+    </template>
+</dom-module>
+<script>
+  (function() {
+    'use strict';
+    /* globals d3 */
+    Polymer({
+      is: 'bar-zoomable-chart',
+      properties: {
+        data: {
```

Comment:
It would be better if, instead of having a single "data" property that contained a "locations" array and a "data" array, this component had two separate properties that took the "locations" (data.locations) and "data" (data.data).  This helps make the component more reusable.